### DISCUSSION TITLE
Are there something wrong with my HRD curve of H.264 bitstream?

### DISCUSSION THREAD

**Discussion by leeluh** [IENT/YUView#468] [Q&A]
Hi there, I want to know why the HRD curve is so strange? My question is: 1. WHY VBV buffer underflow shows a weird status?      2. The stream has 697 frames but YUView just analyzes the first 500 frames. The command I used to encode by x264 is "x264 --bitrate 720 --vbv-maxrate 1440 --vbv-bufsize 1440 --nal-hrd vbr -o output_file.264 --verbose --log-level debug --input-res WxH input_file.yuv 2>>log.txt". It so appreciate that if someone can help me figure out the reason. :)
![截屏2022-07-12 下午8 45 40](https://user-images.githubusercontent.com/30488353/178493139-1a3a8859-a771-40da-a0c5-366b91fc855d.png)

**Comment by ChristianFeldmann** [comment]
Hi! Yes this is the "modern Art" mode of painting the graph. No I am just kidding obviously. My guess is that something with the time calculation is not quite correct. Would it be possible for you to provide the file to me? We could also do this via a private message if you don't want to post the file here.

**Comment by leeluh** [comment]
您好！您发送的邮件我已收到，我会尽快回复。
